openadapt/models.py

```diff
@@ -572,6 +572,31 @@ def to_prompt_dict(self, include_data: bool = True) -> dict[str, Any]:
         else:
             window_dict["state"].pop("data")
         window_dict["state"].pop("meta")
+
+        # Let,
+        # left = window_dict["left"] = x
+        # top = window_dict["top"] = y
+        # width = window_dict["width"] = w
+        # height = window_dict["height"] = h
+
+        # Compute the centroid of the bounding box
+        # Reference: https://stackoverflow.com/a/18932029
```

Comment:
I don't think we need a reference here, this is self explanatory

Reply:
Done here: 5d4bb1c24311d0c6fe387a7b6143342288048cf7